gui-backend/src/test/java/com/rln/gui/backend/implementation/PartyApiImplTest.java

```diff
@@ -0,0 +1,29 @@
+package com.rln.gui.backend.implementation;
+
+import com.rln.gui.backend.model.PartyDTO;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.restassured.RestAssured;
+import io.restassured.common.mapper.TypeRef;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+
+@TestProfile(GuiBackendTest.class)
+@QuarkusTest
+class PartyApiImplTest extends LedgerBaseTest {
+
+  @Test
+  void getMyParty() {
+    PartyDTO expected = new PartyDTO(LedgerBaseTest.BASEURL, LedgerBaseTest.BANK_BIC, LedgerBaseTest.PARTY_ID, LedgerBaseTest.PARTY_NAME);
+
+    PartyDTO result = RestAssured.given()
+        .when().get("/api/parties/me")
+        .then()
+        .statusCode(200)
+        .extract().body().as(new TypeRef<>() {
+        });
+
+    MatcherAssert.assertThat(result, Matchers.is(expected));
```

Comment:
Some thoughts to consider:
- as you inherit from `LedgerBaseTest` you don't need the qualify the static fields like `BASEURL`
- I think `var`s read better
- `as` has a simpler variation, `TypeRef` is necessary only if you have a generic type, e.g. `List<T>`
- you don't need `given` and `when`
- define variables close to their usage. If nothing else, an "expected" thing belongs to the assertion part of test in my opinion

Here's my variation:
```suggestion
    var result = RestAssured
      .get("/api/parties/me")
      .then()
      .statusCode(200)
      .extract().body().as(PartyDTO.class);

    var expected = new PartyDTO(BASEURL, BANK_BIC, PARTY_ID, PARTY_NAME);
    MatcherAssert.assertThat(result, Matchers.is(expected));
```